scripts/install_test_dependencies.sh

```diff
@@ -15,8 +15,19 @@
 
 set -ex
 
-sudo apt-get update
-sudo apt-get install -y php5-cli
+if [ "${CIRCLECI}" != "true" ]; then
+    sudo apt-get update
+    sudo apt-get install -y php5-cli
```

Comment:
I feel like these shouldn't be here. It ruins this script for OSX and some versions of linux. Can we make it more generic?